packages/launchpad/cypress/e2e/config-warning.cy.ts

```diff
@@ -94,8 +93,10 @@ describe('experimentalSingleTabRunMode', () => {
         })`)
     })
 
+    cy.visitLaunchpad()
+
     cy.get('[data-cy-testingtype="component"]').click()
-    cy.get('h1').contains('Initializing Config').should('not.exist')
+    cy.findByTestId('launchpad-Choose a Browser')
```

Comment:
I think the flake here is related to detached DOM actually... which is something we are working on fixing 😎